templates/zerver/help/link-to-a-message-or-conversation.md

```diff
@@ -16,17 +16,20 @@ issue tracker.
 
 !!! warn ""
     This works for all views, including searches.
-    All URLs in Zulip are permalinks.
+    All URLs in Zulip are designed to be shareable.
 
 ## Link to a specific message
 
 {start_tabs}
 
 {!message-actions-menu.md!}
 
-1. Click **Link to conversation**.
+1. Click **Copy link to message**.
 
 {end_tabs}
 
-The copied link will take you to the topic of the relevant message, scrolled
-down to that message.
+This will copy to your clipboard a permanent link to the message,
+displayed in context (E.g. a stream + topic view).
+
+To make it easy to view these links even if you're not caught up,
+these "link to message" views will never mark messages as read.
```

Comment:
maybe rephrase:

"To make it convenient to follow message links even if you're not caught up, they never mark messages as read."